.github/workflows/push-image.yaml

```diff
@@ -38,13 +38,17 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       IMAGE_TAG: ${{ steps.tag.outputs.value }}
+      REF: ${{ steps.ref.outputs.value }}
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - name: Store git tag value to output
         id: tag
         run: echo "value=$(git describe --tags --always --abbrev=7)" >> $GITHUB_OUTPUT
+      - name: Store git ref value to output
+        id: ref
+        run: echo "value=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
```

Comment:
${GITHUB_REF} is ref value.
ex: `refs/heads/feature-branch-1`, `refs/tags/v1.0.0`

`##*/` omits `refs/heads` and `refs/tags`.
${GITHUB_REF##*/} is `feature-branch-1` or `v1.0.0`